meshroom/ui/app.py

```diff
@@ -61,9 +61,13 @@ def __init__(self, args):
 
         parser.add_argument('input', metavar='INPUT', type=str, nargs='?',
                             help='Meshroom project file (e.g. myProject.mg) or folder with images to reconstruct.')
-        parser.add_argument('--project', metavar='MESHROOM_FILE', type=str, required=False,
+        parser.add_argument('-i', '--import', metavar='IMAGES/FOLDERS', type=str, nargs='*',
```

Comment:
Isn't now the arguments a little bit too redundant? 
We can get rid of `input` at this point and just use project and the imports. It makes life easier, it's more readable and less confusing, and we don't need to check for conflicts. And too bad if we break compatibility, we already broke it with photogrammetry.